src/scripts/genXplatEventing.py

```diff
@@ -509,8 +512,8 @@ def generateSanityTest(sClrEtwAllMan,testDir):
                           coreclrpal
                           ${EVENT_PROVIDER_LINKER_OTPTIONS}
                           )
-    """
-    print >>Testinfo, """
+    """, file=Cmake_file, end="")
```

Comment:
instead of using print can you change the usage here and else where to use filehandle.write. So that it much clearer what file the strings are being written. like in https://github.com/dotnet/coreclr/blob/master/src/scripts/genWinEtw.py#L39
https://github.com/dotnet/coreclr/blob/master/src/scripts/genWinEtw.py#L42